jenkins/end_to_end_tests.sh

```diff
@@ -17,7 +17,14 @@ virtualenv venv
 
 pip install -r requirements/base.txt
 
-# Install pages.
+# Install the page objects from the edx-platform repo.
+# Before doing so, we don't need optimizations for lxml,
+# so install it this way which doesn't bother compiling them.
+STATIC_DEPS=true CFLAGS="-O0"  pip install "lxml==3.4.4"
 paver install_pages
+
+# Set the display to the virtual frame buffer (Xvfb)
+export DISPLAY=:1
```

Comment:
@benpatterson 
**tldr:** I looked into it but called it too much feature creep from the current issue.

**longer:** I figured out the root cause with firefox logging, and started putting in the changes to make that an option but although it's pretty easy even that wasn't a gimme. https://github.com/edx/bok-choy/pull/177

BTW I forget what the firefox output was, but IIRC it was something along the lines of "Could not open the display" which was pretty much a flashing neon sign.